content/api/index.html

```diff
@@ -790,10 +790,16 @@
           what, if any, group states to include. Choose from one or more from
           'all', 'alert', 'warn', or 'no data'. Example: 'alert,warn'",
           :default => "None" %>
+          <%= argument 'name', "A string to filter monitors by name",
+          :default => "None" %>
           <%= argument 'tags', "A comma separated list indicating
           what tags, if any, should be used to filter the list of monitors
           by scope",
           :default => "None" %>
+          <%= argument 'monitor_tags', "A comma separated list indicatinig
+          what custom tags/services, if any, should be used to filter the list
+          of monitors",
+          :default => "None" %>
```

Comment:
How about: "A comma-separated list indicating what service and/or custom tags, if any, should be used to filter the list of monitors"